internal/pkg/fixtures/configs/python-transformation-shared-code.json

```diff
@@ -0,0 +1,28 @@
+{
+  "componentId": "keboola.python-transformation-v2",
+  "name": "Python Transformation Shared Code",
+  "configuration": {
+    "parameters": {
+      "blocks": [
+        {
+          "codes": [
+            {
+              "name": "Shared Code Used",
+              "script": [
+                "{{...}}"
+              ]
+            }
+          ],
+          "name": "Block 1"
+        }
+      ]
+    }
+  },
+  "rows": null,
+  "shared_code_id": "...",
```

Comment:
Napojenie shared code <-> transformation <-> variables, je v KBC realizovane pomocou IDs priamo v konfiguracii, pod definovanymi klucmi.

KBC to nevaliduje, riesi to az docker runner (a UI).

V tomto PR tieto IDs NEriešim, je potrebne pridat mechanizmus, ktory by sa dane konfiguracie prepojili pri vytvarani v testoch. To bude obsahom dalsieho PR.

V `internal/pkg/fixtures/configs/*.json` sa teraz nedaju pouzivat ENV premenne + jeden config moze byt aj vo viacerych `branches` , ak sa tak zada v `initial-state.json`.

Ked bude vyriesene, ako vytvorit prepojenia v testoch, ... tak zostane este hlavny problem a to, ze tieto IDs musime dat z konfiguracii prec -> do manifestu ... pretoze takto, ako to je teraz: nemozes zobraz config z ineho projektu a iba ho nakopirovat + zavolat `persist`, lebo nebudu sediet IDs.